src/draive/lmm/helpers.py

```
@@ -0,0 +1,82 @@
+from draive.lmm.types import LMMOutputDecoder, LMMOutputSelection
```

Pylint (3.3.7)

[convention] Missing module docstring

(C0114)